README.md

```diff
@@ -14,112 +14,30 @@
-```
-
-As a region use for example `eu-central-1`.
-
-For creating a connection to a local [AWS testing instance](https://docs.aws.amazon.com/de_de/amazondynamodb/latest/developerguide/DynamoDBLocal.html) use:
-
-```
-CREATE CONNECTION DYNAMO_CONNECTION
-    TO 'http://localhost:8000'
-    USER 'fakeMyKeyId'
-    IDENTIFIED BY 'fakeSecretAccessKey';
-
-```
-
-Before creating a Virtual Schema you need to [create mapping definitions](doc/gettingStartedWithSchemaMappingLanguage.md) and upload them to a BucketFS bucket.
-
-Finally create the Virtual Schema using:
-
-```
-CREATE VIRTUAL SCHEMA DYNAMODB_TEST USING ADAPTER.DYNAMODB_ADAPTER WITH
-    CONNECTION_NAME = 'DYNAMO_CONNECTION'
-    SQL_DIALECT     = 'DYNAMO_DB'
-    MAPPING         = '/bfsdefault/default/path/to/mappings/in/bucketfs';
-```
- 
-
-## First Steps
-
-Start with the [mapping definition example](doc/gettingStartedWithSchemaMappingLanguage.md).
-
-# Documentation
-
-* [Schema mapping language reference](https://exasol.github.io/dynamodb-virtual-schema/schema_doc/index.html)
-* [Schema mapping software architecture](doc/schemaMappingArchitecture.md)
-
-# Limitations
+## Limitations
 
 * In this version the adapter does not support comparisons between two columns
 
-# Logging & Debugging
-
-The following links explain logging and debugging for the Virtual Schema in general:
-
-* [Logging for Virtual Schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_logging.md)
-* [Remote debugging Virtual Schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_debugging.md)
+## Information for Users
 
-For logging and debugging in the test code, things are easier, 
-as most of the setup code is already included in the AbstractExasolTestInterface.
+* [User Guide](doc/user-guide/user_guide.md)
+* [Schema mapping language reference](https://exasol.github.io/dynamodb-virtual-schema/schema_doc/index.html)
 
-The tests automatically redirect the logs from the Virtual Schema to the tests command line output.
+## Information for Developers
 
-For debugging start a debugger on your development machine listening on port `8000` and 
-start the tests with: `-Dtests.debug="virtualSchema"` or `-Dtests.debug="all"`. 
-The last option also starts the debugger for the UDF calls. This will however typically fail,
- as the UDFs run in parallel and therefore only one can connect to the debugger.
- 
-Additionally the test setup can run the VirtualSchema and the UDFs with a profiler. 
-Therefore we use the [Hones Profiler](https://github.com/jvm-profiling-tools/honest-profiler).
-To use it, download the binaries from the project homepage 
-and place the `liblagent.so` in the directory above this projects root.
-Then enable profiling by adding `-Dtests.profiling="true"` to your jvm parameters.
-      
+* [Developers Guid](doc/development/developers_guide.md)
+* [Schema mapping software architecture](doc/schemaMappingArchitecture.md)
```

Comment:
This link is broken. Here you meant `gettingStartedWithSchemaMappingLanguage`? Or do you plan to add the schema mapping architecture document?

Reply:
I see below that schema mapping architecture doc is removed.